src/components/10-atoms/text/ui.test.js

```diff
@@ -0,0 +1,20 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+
+fixture('Text - basic functionality').page(
+  `${host}/iframe.html?id=atoms-text--text-default`
+);
+
+const TAG = 'axa-text';
+const CLASS = '.a-text';
+
+test('should render text', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-text').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+});
```

Comment:
Shouldn't you test for font-sizes, line-heights and what not here?

Reply:
Hmm. Yes. You are #@# right.

Reply:
I added tons of responsive tests that cover font size check, based on our design rules. So this should prevent us from surprised in components by changing font sizes.